### PR TITLE
DFPL-2546 Show Refuge Address- address option for Respondent/Other persons Journeys

### PR DESCRIPTION
**Pull request by chak-shing-lo-justice**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/DFPL-2546

### Change description ###
DFPL-2546 Show Refuge Address- address option for Respondent/Other persons Journeys

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[X] No
```

### CODE REVIEW

File: ccd-definition/ComplexTypes/CareSupervision/2_Others.json [hunk 1]
```diff
@@ -47,18 +47,28 @@
     "LiveFrom": "01/01/2017",
     "ID": "Others",
     "ListElementCode": "addressKnow",
-    "FieldType": "YesOrNo",
+    "FieldType": "FixedRadioList",
+    "FieldTypeParameter": "IsAddressKnowList",
```
Comment (DanCatchpole) [inline ccd-definition/ComplexTypes/CareSupervision/2_Others.json:51]: I guess just a note for testing, maybe worth setting up an old case with a Yes/No answer on the case already, to confirm this works seamlessly!

File: service/src/main/java/uk/gov/hmcts/reform/fpl/enums/IsAddressKnowType.java [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+import lombok.RequiredArgsConstructor;
+
+import java.util.Arrays;
+
+@Getter
+@RequiredArgsConstructor
+public enum IsAddressKnowType {
+    YES("Yes"),
+    NO("No"),
+    LIVE_IN_REFUGEE("LIVE_IN_REFUGE");
```
Comment (DanCatchpole) [inline service/src/main/java/uk/gov/hmcts/reform/fpl/enums/IsAddressKnowType.java:13]: can remove the extra E here on the enum name!

Comment (chak-shing-lo-justice) [reply service/src/main/java/uk/gov/hmcts/reform/fpl/enums/IsAddressKnowType.java]: Haha, missed that one. Well...still need that fromString for converting "Yes" to IsAddressKnowType.YES.

**Review by DanCatchpole — DISMISSED** [review]
lgtm!

**Review by DanCatchpole — APPROVED** [review]
lgtm!

**Pull request merged**